<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewChan</class>
 <widget class="QDialog" name="NewChan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Chan</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#NewChan {
	background: url(:/images/background_connection.png);
}
#NewChan QLineEdit{
	background: url(:/images/send_box_connection.png) transparent no-repeat;
	border:none;
	height:24px;
	color:white;
}
#NewChan QLabel{
	color:white;
}
#NewChan QPushButton {
background:url(:/images/btn.png) transparent no-repeat;
border:none;
width:80px;
height:22px;
}
#NewChan QPushButton:hover {
background:url(:/images/btn_hover.png) transparent no-repeat;
border:0;
width:70px;
height:26px;
}</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="NameLabel">
     <property name="text">
      <string>Chan name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewChan</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewChan</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
